portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,222 @@
+#UTF-8 {
```

Comment:
indentation seems to have been messed up in this file too.

Reply:
fixed